modules/ROOT/pages/streaming-about.adoc

```diff
@@ -271,6 +248,17 @@ increments at 100 objects per increment and allows a maximum buffer size of
  </sfdc:query>
 ----
 
+[[disable_repeatable_streaming]]
+== Disabling Repeatable Streaming
+
+You can disable repeatable streaming through the `non-repeatable-stream` and `non-repeatable-iterable` strategies. The strategy to use depends on the type of stream.
+
+Use this option only if you are certain that there is no need to consume the stream several times and only if you need a very tight optimization for performance and resource consumption. 
+
+When making that decision, consider the following: 
+
+* Although disabling repeatable streaming improves performance, the significance of that gain is proportional to the size of the streams. Unless you are working with large streams, it is likely that you can do something else in your app to provide a better optimization.
+* Disabling repeatable streaming means that the stream is only read once. Keep in mind that your flow might contain components that require consumption of the full stream even if you do not explicitly specify that requirement. Examples include the Cache component (`<ee:cache>`), some transformations through the Transform Message component (`<ee:transform>`, the For Each component (`<foreach>`) when iterating through a JSON array, and any other component that is using an expression that accesses the stream. 
```

Comment:
`and any other component that is using an expression that accesses the stream. ` -> `and any other component that accesses the stream directly or through an expression`